site/docs/v1/tech/apis/passwordless.adoc

```diff
@@ -68,6 +68,9 @@ include::docs/src/json/passwordless/start-request.json[]
 |401
 |You did not supply a valid Authorization header. The header was omitted or your API key was not valid. The response will be empty. See link:/docs/v1/tech/apis/authentication[Authentication].
 
+|404
+|The user was not found.
```

Comment:
We have some existing messaging for a `404` that we can re-use or mostly re-use. 

```
The user was not found. The response will be empty.
```